cross/7za/PLIST

```diff
@@ -0,0 +1 @@
+bin:bin/7za
```

Comment:
For some reason SABnzbd isn't seeing it and jumping to the old system 7zip.
```
2025-01-13 22:37:33,269::INFO::[SABnzbd:460] UNRAR binary... found (/volume1/@appstore/sabnzbd/bin/unrar)
2025-01-13 22:37:33,269::INFO::[SABnzbd:468] UNRAR binary version 7.10
2025-01-13 22:37:33,270::INFO::[SABnzbd:475] 7za binary... found (/bin/7z)
2025-01-13 22:37:33,270::INFO::[SABnzbd:477] 7za binary version 16.02
```

We just look at the `PATH` and then check for any of the listed executable names in each part of the `PATH`.
https://github.com/sabnzbd/sabnzbd/blob/7de08693765fac39192e4cbcffdcc725244a943e/sabnzbd/newsunpack.py#L148-L154

Reply:
@Safihre thanks for the feedback, guess I fixed it.